src/containers/Demo/index.tsx

```diff
@@ -1,57 +1,37 @@
-import React from 'react';
-import { connect } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 
 import Loader from 'src/components/Loader';
 import { getDemoData, getMockData } from 'src/state/demo/actions';
+import AccordionList from './AccordionList';
 
-interface IDemoDispatchProps {
-  fetchDemoData: () => void;
-  fetchMockData: () => void;
-}
+const Demo: React.FC = () => {
+  const { loading, posts, mockPosts } = useSelector((state: ReduxStore) => ({
+    posts: state.demo.data,
+    loading: state.common.demoLoading,
+    mockPosts: state.demo.mockData,
+  }));
 
-interface IDemoStateProps {
-  posts: DemoData[];
-  loading: boolean;
-  mockPosts: DemoData[];
-}
+  const dispatch = useDispatch();
 
-type IDemoProps = IDemoDispatchProps & IDemoStateProps;
+  useEffect(() => {
+    dispatch(getDemoData());
+    dispatch(getMockData());
+    /* tslint:disable-next-line */
```

Comment:
Why do we need this? We are not using any such rule

Reply:
We have added alignment rule for arguments in tslint.json, which results in `arguments are not aligned` error.